website/docs/cloud-docs/api-docs/notification-configurations.mdx

```diff
@@ -112,6 +112,65 @@ For Slack and Microsoft Teams notifications, the payload conforms to the respect
 }
 ```
 
+### Workspace Notification Payload
+
+| Name                                         | Type   | Description                                                               |
+| -------------------------------------------- | ------ | ------------------------------------------------------------------------- |
+| `payload_version`                            | number | Currently always "2".                                                     |
+| `notification_configuration_id`              | string | ID of the configuration which defined this notification.                  |
+| `notification_configuration_url`             | string | URL of the Notification Configuration.                                    |
+| `trigger_scope`                              | string | "assessment" for workspace notifications.                                 |
```

Comment:
I don't know what we mean by "assessment" here - help :-)

Reply:
This field is intended to support more notification types in v2 in the future.  Drift detection notifications will all have the value `"assessment"`.  If we migrated run notifications to v2, they would have the value `"run"`.  Hypothetical future notification types would have a new value.  But for now, only drift detection notifications use v2, so this field (**for now**) will only even have the value `"assessment"`.  I very much do not like what I came up with here, so please give me better words :)